app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -132,17 +137,12 @@ public void onDrawerStateChanged(int newState) {
         }
 
         checkInformedConsent();
+    }
 
-        // XXX: This is probably not correct: we would like to send
-        // info to the orchestrator only when the network or any other
-        // orchestrator parameter like country code changed.
-        //TODO remove this code in 1.1.6
-        String token = FirebaseInstanceId.getInstance().getToken();
-        if (token != null) {
-            NotificationService ns = NotificationService.getInstance(this);
-            ns.setDevice_token(token);
-            ns.sendRegistrationToServer();
-        }
```

Comment:
FTR, we verified with @lorenzoPrimi that this was causing two back-to-back orchestration requests.